BUILD.bazel

```diff
@@ -1,5 +1,6 @@
 load(":toolchain.bzl", "pandoc_toolchain")
-load(":pandoc.bzl", "pandoc")
+
+exports_files(["README.md", "LICENSE"])
```

Comment:
What's up with this change specifically? Do you need to access/include these files externally?

Reply:
Let's say yes:
- I moved your samples from `//` to `//samples`. This allows to generate the readme in all formats with `blaze build //samples/...`. If you perform `bazel build //...` then you download pandoc for all platforms.
- The `LICENSE` file should always be exported in my opinion. Client projects which depend on you should include the LICENSE file in their notice; and that can be automated if the LICENSE is exported.

Reply:
Isn't the preferred way of exporting license information to use the `licenses` arguments provided by many of the functions?

https://docs.bazel.build/versions/master/be/common-definitions.html#common.licenses

Reply:
I'll resolve because because it is off-topic.
Usually, we indicate `license(["notice"])`, a generic type that regroups all licenses that require a notice declaration.
But the third-party client still has the responsibility to show this notice, and it's usually done by showing the respective licenses of its dependencies.